src/containers/App/index.js

```diff
@@ -61,16 +61,15 @@ class App extends Component {
 }
 
 App.propTypes = {
-  account: PropTypes.string,
+  activeProvider: PropTypes.string,
   blockchainConnection: PropTypes.bool,
   children: PropTypes.node,
-  connectBlockchain: PropTypes.func,
   location: PropTypes.object,
   hasWallet: PropTypes.bool,
 }
 
 const mapStateToProps = state => ({
-  account: state.blockchain.defaultAccount,
+  activeProvider: state.blockchain.activeProvider,
```

Comment:
Why is in other components  `getSelectedProvider` selector used and in this one a direct state access?